### ISSUE TITLE
etherape permission denied

### ISSUE THREAD

**Issue by visious25** [BlackArch/blackarch#3577]
Hi, I'm getting this in etherape: Capture > Interfaces (In eth0, wlan0 and any)

you don't have permission to capture on that device (socket operation not permitted)
and various messages to the others.

Also I'm getting the same message: Capture > Mode > TCP/IP

**Comment by noraj** [comment]
`sudo etherape` or `sudo -E etherape`

**Comment by visious25** [comment]
Yes, "sudo etherape or sudo -E etherape" working in terminal. I'm asking you if there's a way to change command box in Menu > Internet > EtherApe?

**Comment by noraj** [comment]
You can modify the command in the target [Desktop Entry](https://wiki.archlinux.org/title/Desktop_entries).

**Comment by visious25** [comment]
Hi, I tried several commands but still I have no luck. I have edited both Desktop Enty 

Exec=pkexec -u root etherape
Exec=pkexec -u root /usr/bin/etherape
Exec=pkexec etherape
Exec=pkexec /usr/bin/etherape

and Menu Enty

pkexec -u root etherape
pkexec -u root /usr/bin/etherape
pkexec etherape
pkexec /usr/bin/etherape

Can you please tell me what should I do?

**Comment by visious25** [comment]
No, this command is working in terminal well, but still not working in Desktop Enty and  Menu Enty I have edited both.

/usr/share/applications/etherape.desktop
Exec=pkexec env DISPLAY=$DISPLAY XAUTHORITY=$XAUTHORITY etherape
Exec=pkexec env DISPLAY=$DISPLAY XAUTHORITY=$XAUTHORITY /usr/bin/etherape
Menu > Internet > EtherApe > Edit Application...
pkexec env DISPLAY=$DISPLAY XAUTHORITY=$XAUTHORITY etherape
pkexec env DISPLAY=$DISPLAY XAUTHORITY=$XAUTHORITY /usr/bin/etherape
